### DISCUSSION TITLE
Nightly updates

### DISCUSSION THREAD

**Discussion by DarkShyMW** [mastodon/mastodon#19549] [General]
Hello! I think - of you can create nightly updates - it can help your with bug reporting.

**Comment by Tslmuun** [comment]
git remote add  nightly https://https://github.com/mastodon/mastodon.git

git fetch nightly
git checkout latest commits

**Reply by DarkShyMW** [reply]
mastodon@server03:~/live$ git checkout latest commits
error: pathspec 'latest' did not match any file(s) known to git
error: pathspec 'commits' did not match any file(s) known to git

**Comment by Tslmuun** [comment]
![Screenshot_20221031-063324](https://user-images.githubusercontent.com/50679343/198904881-014bb2e9-7df9-44ed-beda-035ce15a69d8.png)

git checkout 26478f461cafc42d39bc0baf235989e1df7ee0bb

**Comment by Tslmuun** [comment]
Please operate in a test environment！
